### ISSUE TITLE
Make Sidekick panel resizable

### ISSUE THREAD

**Issue by halohalospecial** [halohalospecial/atom-elmjutsu#10]
User should be able to drag the edge of the panel to resize it, similar to that of the `linter`'s error panel.

**Comment by OvermindDL1** [comment]
Yeah just hit that, when the cursor happens to hit near `elm-mdl`'s `Material.Tooltip` call the sidekick becomes my *entire* screen, of which I accidentally clicked inside of, thus focusing the sidekick, thus I could not get it to go away without killing it via command.  ^.^

**Comment by mbylstra** [comment]
I keep getting stuck in a state where I can't do anything because the sidekick is filling the entire screen due to a long doc. I then have to completely quit Atom, losing any unsaved work. Is there a key combo to close or disable the sidekick if this happens?

Setting the sidekick position to right instead of the default position of bottom makes it not fill the entire screen, but puts it in an inconvenient location. Of course having a resizable panel would be ideal - I'm just looking for a quickfix for this particular problem.

**Comment by mbylstra** [comment]
The same thing can happen with the sidekick at the right position, if you have record with a lot of fields.

**Comment by mbylstra** [comment]
A workaround is to use the keyboard shortcut for closing the current window: `Ctrl+w` in Windows and Linux,  and `Cmd+w` on Mac.

**Comment by halohalospecial** [comment]
Hi!  You can do one of the ff:

- Execute `Elmjutsu: Toggle Sidekick` from the command palette.

- Bind `elmjutsu:toggle-sidekick` to a key combo in your ~/.atom/keymap.cson.  For example:
```
'atom-workspace':
  'f1': 'elmjutsu:toggle-sidekick'
```

- Configure `Sidekick Size` in the package settings so that the height (or width) will remain constant.

**Comment by mbylstra** [comment]
Hi. All of those options are good. Thanks for the help!

**Comment by gampleman** [comment]
Perhaps it could be turned into a Dock?

**Comment by halohalospecial** [comment]
Great idea, @gampleman !
